utils/get_prometheus_data.py

```diff
@@ -0,0 +1,145 @@
+import os
+import requests
+import json
+import logging
+import urllib3
+from datetime import datetime, timedelta
+import time
+import sys
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+
+logger = logging.getLogger("snafu")
+
+class get_prometheus_data():
+    def __init__(self, action):
+
+        self.uuid = action["uuid"]
+        self.user = action["user"]
+        self.cluster_name = action["cluster_name"]
+        self.test_config = action["test_config"]
+        self.start = action["starttime"]
+
+        # change datetime in seconds string to datetime object
+        endtime_object = datetime.fromtimestamp(int(action["endtime"]))
+        # add 120s buffer to end time
+        endtime_object = endtime_object + timedelta(seconds=120)
```

Comment:
what happens if resulting timestamp is in the future?  Can this happen?

Reply:
No sure of the likelihood of this happening, but query will still be performed, if no data is there nothing is returned for the future time. AFAIK that's what I have observed, could be wrong.

Reply:
OBE